packages/core/src/screens/main/hooks/use-customer-name-format/helpers.test.ts

```
@@ -0,0 +1,251 @@
+/**
+ * @jest-environment node
+ */
+import { extractNameFromJSON } from './helpers';
+import type { JSON } from './helpers';
```

Biome (2.3.13)

[error] Do not shadow the global "JSON" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)